enhancements/network/on-prem-dual-stack-vips.md

```diff
@@ -0,0 +1,339 @@
+---
+title: dual-stack-vips
+authors:
+  - @cybertron
+reviewers:
+  - @creydr
+approvers:
+  - @patrickdillon
+  - @kirankt
+  - @shardy
+  - someone from MCO
+api-approvers:
+  - ???
+creation-date: 2022-03-01
+last-updated: 2022-03-02
+tracking-link:
+  - https://issues.redhat.com/browse/SDN-2213
+see-also:
+  - "/enhancements/on-prem-service-load-balancers.md"
+replaces:
+superseded-by:
+---
+
+# Dual Stack VIPs
+
+## Summary
+
+Originally the on-prem loadbalancer architecture supported only single-stack
+ipv4 or ipv6 deployments. When dual stack support was added for the baremetal
+platform, we carried over the VIP configuration that only allowed for a single
+VIP to be specified. The thinking was that in dual stack clusters every node
+would have access to either ipv4 or ipv6 endpoints. While this is true of the
+nodes, it is not necessarily true of the applications that run in the cluster
+and may need to interact with the API or ingress services.
+
+This design is proposing the addition of a second VIP for dual stack clusters
+so API and ingress are accessible from either ip version.
```

Comment:
cc @danwinship for API server dual-stack https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/2438-dual-stack-apiserver